autogpt/configurator.py

```diff
@@ -82,10 +82,22 @@ def create_config(
 
     if gpt3only:
         logger.typewriter_log("GPT3.5 Only Mode: ", Fore.GREEN, "ENABLED")
+        CFG.set_smart_token_limit(CFG.fast_token_limit)
+        if CFG.use_azure:
+            CFG.set_azure_deployment_id_for_model(
+                CFG.smart_llm_model,
+                CFG.get_azure_deployment_id_for_model(CFG.fast_llm_model),
```

Comment:
This swap feels unexpected, although I can see the rationale for wanting it to pull the GPT-3.5 model.

Instead, I'd fix line 91 to do what the user is asking:

If gpt3only: Set all LLM models to GPT3 explicitly
=> Set fast-model = GPT3 
=> Set smart-model = GPT3

Then, I think get_azure_deployment_id_for_model() will already work because it returns fast for fast and smart for smart—no swapping needed. Am I reading this wrong?